src/filters/colormatrix/ColorMatrixFilter.js

```diff
@@ -575,6 +577,26 @@ export default class ColorMatrixFilter extends core.Filter
     {
         this.uniforms.m = value;
     }
+
+    /**
+     * The interpolation value to use when mixing the original and resultant colors.
+     *
+     * When the value is 0, the original color is used without modification.
+     * When the value is 1, the result color is used.
+     * When in the range (0, 1) the color is interpolated between the original and result by this amount.
+     *
+     * @member {number}
+     * @default 1
+     */
+    get interpolation()
```

Comment:
I'm still hung-up on the verbosity of "interpolation". It's not really a term we use elsewhere in PIXI. And, in general, I love to avoid more technical terms like this. I think it makes the API more accessible. TBH, I think this is `alpha` or `opacity` and makes it easier to understand what changing that value would do. 

Does anyone else feel strongly about this?